src/main/java/com/example/demo/controller/RecordController.java

```diff
@@ -18,6 +18,7 @@ public class RecordController {
 
     @Autowired
     private RecordService recordService;
+    private int displayNumber = 0;
```

Comment:
Not sure what is it for, but I'm pretty sure it might not work as expected - `Controller` is a singleton and whenever there comes a request it is possible to change it any way one wants.